README.md

```diff
@@ -139,6 +141,16 @@ host: localhost
 #profile_host: 127.0.0.1
 #profile_port: 7070
 
+# If specified, the HTTP read timeout for a client request in seconds.
+# (applies to the frontend not the proxy backend)
+# Time units can be one of: "s", "m", "h".
+#http_read_timeout: 5s
+
+# If specified, the HTTP write timeout for a server response in seconds.
+# (applies to the frontend not the proxy backend)
+# Time units can be one of: "s", "m", "h".
```

Comment:
I think we should skip this mostly duplicated text, and refer to "HTTP read/write timeouts" in the first sentence, with no newline between the two example lines. And reuse the same sentence in brackets as suggested for the flag text.

Reply:
fixed !